.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base"],
+  "rules": {
+    "no-shadow": "off",
+    "no-param-reassign": "off",
+    "eol-last": "off",
+    "import/extensions": [
+      1,
+      {
+        "js": "always",
+        "json": "always"
+      }
+    ]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```

Comment:
Nice work adding the eslint file, but I think you still haven't installed eslint on your project. currently, eslint is not present in your checked linters. kindly make the necessary adjustment.😉😉😉
![linters_reqr](https://user-images.githubusercontent.com/64643995/215579583-d7417ac5-8d33-47ad-a9d6-0a766458bddd.png)

![Screenshot from 2023-01-30 20-41-50](https://user-images.githubusercontent.com/64643995/215581150-c2b13650-50ea-441a-84b6-be2ea4e6c99b.png)

Reply:
Thanks for the review @Ibizugbe